src/windows/about.py

```diff
@@ -396,9 +389,7 @@ def __init__(self):
         github_html = '''
             <html><head/><body>
             <p align="center">
-                <a href="https://github.com/OpenShot/">
-                <span style=" text-decoration: underline; color:#55aaff;">%s</span>
-                </a>
+                <a href="https://github.com/OpenShot/"><span style="text-decoration: underline; color:#55aaff;">%s</span></a>
```

Comment:
Again. I don't think we need all this in one line.

But if you'd like to save some space. You could definitely make a css class in one of the `.css` filese for all this repeated style info.

Reply:
Same af the other link might end up being:

                 <a href="https://github.com/OpenShot/">%s</a>